### ISSUE TITLE
Progress component rounding causes jumpy animation

### ISSUE THREAD

**Issue by trentjones21** [petalframework/petal_components#313]
https://github.com/petalframework/petal_components/blob/39a2b5f879266e85abb8bdbb8075b99a7c233b1a/lib/petal_components/progress.ex#L22

I'm animating a Progress component, and I'm wondering if it's necessary to `round` the width percentage of the filled in portion of the bar.  This causes the animation to be extremely jumpy when going from one full percentage point to the next.  

Perhaps animation is outside the intended scope of this component, but would it hurt to either remove the `round` function or round it to a one or two decimal places, as opposed to whole numbers?

**Comment by mplatts** [comment]
Happy for it to be removed if it helps with performance and no negative effects. Not sure why we do it. Would you be able to do a PR?

**Comment by trentjones21** [comment]
Yep! Here you go: https://github.com/petalframework/petal_components/pull/314

**Comment by mplatts** [comment]
This was fixed in a807423 — which rounds to 2 decimal places instead of whole numbers. Fittingly, the fix was authored by you @trentjones21! 

Released in v2.0.0. Closing as resolved.
